src/core/runtime/preload-browser.js

```diff
@@ -1,27 +1,25 @@
 /* eslint-env browser */
 'use strict'
 
+const { default: PQueue } = require('p-queue')
+const { default: ky } = require('ky-universal')
 const debug = require('debug')
 
 const log = debug('ipfs:preload')
 log.error = debug('ipfs:preload:error')
 
+// browsers limit concurrent connections per host,
+// we don't want preload calls to exhaust the limit (~6)
+const _httpQueue = new PQueue({ concurrency: 4 })
```

Comment:
Why the underscore here & in the `runtime/dns-browser.js` file?

Reply:
refactor scar tissue, fixed in https://github.com/ipfs/js-ipfs/pull/2304/commits/2e82b2a9b468db99937256f3d77261eaef9a8c0f